docs/en/UI/Angular/HTTP-Error-Handling.md

```diff
@@ -0,0 +1,195 @@
+
+**In Summary**
+
+- Services are sorted by their priority number.
+- Start from highest priority service and run canHandle() method. Pick the service if can handle the error, if not check next service.
+- If the service found, run the execute method of a service. Done.
+
+See an example:
+
+```ts
+// custom-error-handler.service.ts
+import { inject, Injectable } from "@angular/core";
+import { HttpErrorResponse } from "@angular/common/http";
+import { CustomHttpErrorHandlerService } from "@abp/ng.theme.shared";
+import { CUSTOM_HTTP_ERROR_HANDLER_PRIORITY } from "@abp/ng.theme.shared";
+import { ToasterService } from "@abp/ng.theme.shared";
+
+@Injectable({ providedIn: "root" })
+export class MyCustomErrorHandlerService
+  implements CustomHttpErrorHandlerService
+{
+  // You can write any number here, ex: 9999
+  readonly priority = CUSTOM_HTTP_ERROR_HANDLER_PRIORITY.veryHigh;
+  private toaster = inject(ToasterService);
+  private error: HttpErrorResponse | undefined = undefined;
+
+  // What kind of error should be handled by this service? You can decide it in this method. If error is suitable to your case then return true; otherwise return false.
+  canHandle(error: unknown): boolean {
+    if (error instanceof HttpErrorResponse && error.status === 400) {
+      this.error = error;
+      return true;
+    }
+    return false;
+  }
+
+  // If this service is picked from ErrorHandler, this execute method will be called.
+  execute() {
+    this.toaster.error(
+      this.error.error?.error?.message || "Bad request!",
+      "400"
+    );
+  }
+}
+```
+
+```ts
+
+// app.module.ts
+import { CUSTOM_ERROR_HANDLERS, ... } from '@abp/ng.theme.shared';
+import { MyCustomErrorHandlerService } from './custom-error-handler.service';
+
+@NgModule({
+  // ...
+  providers: [
+    // ...
+    { provide: CUSTOM_ERROR_HANDLERS, useExisting: MyCustomErrorHandlerService, multi: true }
+  ]
+})
+export class AppModule {}
+```
+
+In the example above:
+
+- Created a service named `MyCustomErrorHandlerService`, and provided via `useExisting` key because we dont want another instance of it. And set `multi` key to true because ABP default error handlers are also provided with **CUSTOM_ERROR_HANDLERS** injection token.
+
+- 400 errors are handled from custom `MyCustomErrorHandlerService`. When a 400 error occurs, backend error message will be displayed as shown below:
+
+![custom-error-handler-toaster-message](images/custom-error-handler-toaster-message.jpg)
+
+**Note 1:** If your service cannot handle the error. Then ABP will check the next Error Service.
+**Note 2:** If none of the service handle the error. Then basic confirmation message about the error will be shown to the user.
+**Note 3:** You can provide more than one service, with CUSTOM_ERROR_HANDLER injection token.
+**Note 4:** If you want your custom service to be evaluated (checked) earlier, set the priority variable high.

```

Comment:
**Do notes like below please**

### Notes
- If your service cannot handle the error. Then ABP will check the next Error Service
- If none of the service handle the error. Then basic confirmation message about the error will be shown to the user
- You can provide more than one service, with CUSTOM_ERROR_HANDLER injection token
- If you want your custom service to be evaluated (checked) earlier, set the priority variable high